src/style.css

```diff
@@ -0,0 +1,85 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", serif;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  height: 60px;
+  padding: 0 50px;
+  justify-content: space-between;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  align-self: center;
+  width: 40px;
+  height: 40px;
+
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  align-items: center;
+  list-style: none;
+}
+
+.nav__item {
+  display: flex;
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  padding: 60px;
```

Comment:
```suggestion
  height: 60px;
```

Please, read more carefully my comments. You should have changed `height` not `padding`:
![1](https://user-images.githubusercontent.com/66570142/113253940-a0957600-92ce-11eb-8307-8a205ad71edd.png)

Now your page looks really weird
![2](https://user-images.githubusercontent.com/66570142/113253973-a8edb100-92ce-11eb-82b2-61ff29902a5a.png)